device/demo/ui/dd_default.tscn

```diff
@@ -149,13 +151,17 @@ anchor_left = 0.0
 anchor_top = 0.0
 anchor_right = 0.0
 anchor_bottom = 0.0
+margin_right = 1628.0
+margin_bottom = 229.0
 rect_pivot_offset = Vector2( 0, 0 )
+rect_clip_content = false
 mouse_filter = 1
 mouse_default_cursor_shape = 0
-size_flags_horizontal = 2
-size_flags_vertical = 2
+size_flags_horizontal = 3
+size_flags_vertical = 3
 custom_constants/separation = 10
 alignment = 0
+_sections_unfolded = [ "Size Flags" ]
```

Comment:
I think that maybe we should make a habit of closing sections before staging scene files for commits, and maybe add this to the contributing guidelines as well, since it adds noise to the diff. Not really a problem in this case though, since there are already the other diffs, like the node indices that Godot 3.0 insists on adding.